packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.scss

```diff
@@ -136,6 +136,8 @@ $SearchBox-sidePadding: 8px; // padding in input on left and right sides without
     &.is-expanded {
       @media screen and (-ms-high-contrast: active) {
         @include highContrastBorder($ms-color-white);
+        margin-left: -1px; 
```

Comment:
What issue is this fixing? I'm guessing the problem is in the `highContrastBorder()` mixin and that we shouldn't be using negative margins here.